docs/source/_tags/bqskit.md

```diff
@@ -0,0 +1,10 @@
+(sphx_tag_bqskit)=
```

Comment:
Most of these added files in `_tags` are causing a few build warnings. Might need to downgrade sphinx version. 

`RemovedInSphinx90Warning: Sphinx 9 will drop support for representing paths as strings. Use "pathlib.Path" or "os.fspath" instead`

https://github.com/unitaryfund/mitiq/actions/runs/10363857000/job/28688193465?pr=2467#step:8:212

Reply:
This warning appears to be an issue that could be fixed by downgrading Sphinx < 8.0.0

https://github.com/executablebooks/MyST-NB/issues/619